airlock/settings.py

```diff
@@ -209,16 +210,22 @@ def run_connection_init_queries(*, connection, **kwargs):
 
 USE_TZ = True
 
+# Get the location of the built docs from the mkdocs config
+# The build docs dir is added to STATICFILE_DIRS so that we
+# can serve it within airlock
+with (BASE_DIR / "mkdocs.yml").open() as mkdocs_config:
+    DOCS_SITE_DIR = yaml.load(mkdocs_config, Loader=yaml.Loader)["site_dir"]
```

Comment:
Not suggesting this needs changing but just mentioning an alternative approach here which I've only recently started using: you could hardcode the value here to just:
```py
DOCS_DIR = BASE_DIR / "docs"
```

Then you have tiny test (maybe added to `tests/integration/test_settings.py`) which loads and parses `mkdocs.yaml` and checks that the configuration matches what's in settings.

This prevents us changing the config in one place and forgetting to change it in another, but without introducing any additional runtime complexity (admittedly not much in the case, but still it would be nicer not to have to load and parse a yaml file in settings.)

Obviously the disadvantage is that it doesn't _automatically_ stay in sync, but for values which we don't expect to change very frequently I don't think that matters too much.

I mention this more for the sake of spreading the pattern around than because I think it's particularly important to apply it in this cas.

Reply:
Ah - I had it hard coded before (it would be `BASE_DIR` / "mkdocs_build") but changed it to this...I'd forgotten about the settings test